integration/nemo/examples/peft/code/megatron_gpt_peft_fl_eval_config.yaml

```diff
@@ -59,7 +59,7 @@ model:
   ffn_dropout: 0.0
 
   peft:
-    peft_scheme: "ptuning"  # can be either adapter,ia3, or ptuning
+    peft_scheme: "lora"  # can be either adapter,ia3, or ptuning
```

Comment:
suggest enhance comment to `# can be either lora, adapter, ia3, or ptuning`

Reply:
Right. Lora is missing. Added.